packages/webdriverio/src/commands/element/click.ts

```diff
@@ -86,7 +86,7 @@ export default async function click(
     options?: ClickOptions
 ) {
     if (typeof options === 'undefined') {
-        return this.elementClick(this.elementId)
+        options = {}
```

Comment:
I don't think we can consider this as a solution. The original idea here is to use the standard WebDriver click if no special options are passed in. Errors such as `element not interactable` are legit issues and should be investigated. The reason why you don't get this when using the actions API is because these commands are not connected to an element and therefor those "intractability" checks aren't made.

Reply:
Okay cool.
Let me work on another solution then that would cover all the scenarios.

Reply:
The one (`click({})`) might just not throw an error, can you verify if the click at least works? I wonder why the element is not intractable. As last option I could imagine to retry the click using the action command instead.

Reply:
Okay, so what I did now was just to add the scrollIntoView before the click.
Reason is that the rest of the code does the scroll to.
If that's not the preferred way of doing it, then it's fine, but I wonder why the other command actually do it then.

Reply:
The WebDriver protocol already defines that for every click the browser should scroll the element into viewport. But it only scrolls it to the edge of the viewport and it might help to scroll it into the center of it.

Did you test the new behavior with your use case?

Reply:
Yeah, works perfectly.

Can also raise it to v7 branch if you're happy with it.

Reply:
If you add a unit test that would be great. And yes, the same pr to `v7` helps to release it to the current `stable` version.

Reply:
>I did now was just to add the scrollIntoView before the click.

To be honest I am not happy about this. You add unnecessary scroll for hundreds of thousands users who don't need it.
For me `$().click()` scrolls s webdriver spec defines. Seems it will be better if you cover your corner case by custom click command in your own project or raise an issue in chrome\geckodriver repo.